src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <title>To Do List</title>
+    <script src="https://kit.fontawesome.com/f6777e9957.js"></script>
+    <script src="../modules/addTask.bundle.js"></script>
+    <script src="../modules/remove.bundle.js"></script>
+</head>
+
+<body>
+    <ul class="toDo">
+        <li class="table">
+            <h4>To Do List</h4>
+        </li>
+        <li class="table">
+            <input type="text" id="add" placeholder="Add tasks">
+            <input type="hidden" id="saveIndex" />
+        </li>
+        <li class="table">
+            <ul id="tasks">
+
+            </ul>
+        </li>
+        <li class="table btn">
+            <button type="button" id="clear">Clear completed tasks</button>
+        </li>
+    </ul>
```

Comment:
I think that it would be better to enclose your list inside a semantic tag such as "section" for better code readability and presentation

Reply:
Hey @kemigabocatherine, for the last suggestion I think it would be better to live it like that. Because I didn't use semantic tags in my HTML, however, I put lists inside the `ul` container as suggested in the project's requirements

What do you think about that?

Reply:
I think that it would be better to enclose the whole `ul` in a container since the `ul` is the `unordered list` and the `li` are `list items` hence the `ul ` is not really a container